CryptoAnalysis/.project

```diff
@@ -10,6 +10,11 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
Please revert the changes in this file.

Reply:
I did it. Thanks for noticing that. Maybe that is why my previous Shippable build was failing